app/DAO/BakeryDB.scala

```diff
@@ -0,0 +1,92 @@
+package DAO
+
+import doobie._
+import doobie.implicits._
+import cats.effect.IO
+import doobie.postgres.implicits._
+import models.Product
+import cats.effect.unsafe.implicits.global
+import play.api.Configuration
+import play.api.db.Database
+
+import java.time.LocalDateTime
+import java.util.UUID
+import javax.inject._
+
+class BakeryDB @Inject() (config: Configuration, db: Database) {
+
+  val xa = Transactor.fromDriverManager[IO](
+    config.get[String]("db.default.driver"),
+    config.get[String]("db.default.url"),
+    config.get[String]("db.default.username"),
+    config.get[String]("db.default.password")
+  )
+
+  def addProduct(
+      name: String,
+      quantity: Int,
+      price: Double
+  ): Int = {
+    val createdAt = LocalDateTime.now()
+    val insertStatement = {
+      sql"INSERT INTO product (id, name, quantity, price, created_at, updated_at) VALUES (gen_random_uuid(), $name, $quantity, $price, $createdAt, $createdAt)"
+    }
```

Comment:
is there a way to instantiate a case class and then not have to specify each value to be updated here individually?